exec/java-exec/src/main/java/org/apache/drill/exec/ExecConstants.java

```diff
@@ -680,4 +687,13 @@ public static String bootDefaultFor(String name) {
 
   public static final String ALLOW_LOOPBACK_ADDRESS_BINDING = "drill.exec.allow_loopback_address_binding";
 
+  /** Enables batch size statistics logging */
+  public static final String STATS_LOGGING_BATCH_SZ_OPTION = "drill.exec.stats.logging.batch_sz";
```

Comment:
Usually the amount of logging is configured using logger level.

Reply:
We are using both:
- Record batch will automatically show up when debug level is used
- or we explicitly this configuration; this basically help us minimize the amount of logging
- Will also provide the ability to enable record batch stats per operator type

Reply:
Please point me to other places where Drill uses both log level and configuration settings to configure the amount of logging.